samples/matter/light_bulb/boards/nrf54lm20dk_nrf54lm20a_cpuapp.overlay

```diff
@@ -10,10 +10,10 @@
 	};
 
 	aliases {
-		// Configure PWM module for led1 (LED2 on the board)
+		/* Configure PWM module for led1 (LED2 on the board) */
```

Comment:
why once online comment is set by `/* <comment> */` and once `// <comment>` ?

Reply:
they all should be `/* */`